src/utilities/plugin-utils.ts

```diff
@@ -42,15 +41,23 @@ export const findUsedApprovedScripts = (activities: Activity[]) => {
           if (!activity.pages[page].sections[section].is_hidden) {
             for (let embeddableNum = 0; embeddableNum < activity.pages[page].sections[section].embeddables.length; embeddableNum++) {
               const embeddable = activity.pages[page].sections[section].embeddables[embeddableNum];
-              if (embeddable.type === "Embeddable::EmbeddablePlugin" && embeddable.plugin?.approved_script_label === "teacherEditionTips" && teacherEditionMode) {
+              // NOTE: TODO this needs to be fixed.
+              //This change might cause there to be extra space when running in non teacher edition mode and viewing a page with
+              // with teacher edition items on it.
+              //The issue is that the teacher edition item wrapper will be added to the page, and then the teacher edition plugin inside
+              //of it will decide not render anything because the app isn't in teacher edition mode.
+              //That behavior by itself is correct because we are trying to not have these "modes" be explicitly part of the
+              //activity player (or lara runtime). But the problem is that the item wrapper usually has some padding or margins.
+              //So even though it has no real content, the padding will still show up in the runtime.
+              //This change was from master when new-sections was rebased on it.
```

Comment:
Sorry now that I understand this better, this comment should not be added. The extra space does not happen in master which also doesn't have this explicit teacher edition check. 

It works because there is another check in this component with a similar comment:
https://github.com/concord-consortium/activity-player/blob/67f77441b5f54773a4984e9fceb6cc52fa9e3e55/src/components/activity-page/embeddable.tsx#L95-L100